index.bs

```diff
@@ -1421,6 +1422,10 @@ many [=aggregatable attribution reports=] can be in the [=aggregatable attributi
 controls how many bits of information can be exposed associated with a single [=attribution source=].
 The keys are «[=source type/navigation=], [=source type/event=]». The values are non-negative doubles.
 
+<dfn>Max event-level attribution scope information gain per source</dfn> is a [=map=] that
```

Comment:
Why does this use the term `information gain` when the preceding value uses `channel capacity`?

Reply:
And also related, for the corresponding debug type name.

Reply:
Nice catch. changed `information gain` to `channel capacity` for consistency.